source/reference/s3-api-compatibility.rst

```diff
@@ -0,0 +1,155 @@
+====================
+S3 API Compatibility
+====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+This page documents all S3 APIs supported by MinIO Object Storage.
+For reference documentation on any given API, see the corresponding documentation for Amazon S3.
+
+.. important::
+
+   MinIO strongly recommends using an :ref:`S3-Compatible SDK <minio-drivers>` for performing object storage operations.
+
+Object APIs
+-----------
+
+- :s3-api:`CopyObject <API_CopyObject.html>`
+- :s3-api:`DeleteObject <API_DeleteObject.html>`
+- :s3-api:`DeleteObjects <API_DeleteObjects.html>`
+- :s3-api:`DeleteObjectTagging <API_DeleteObjectTagging.html>`
+- :s3-api:`GetObject <API_GetObject.html>`
+- :s3-api:`GetObjectAttributes <API_GetObjectAttributes.html>`
+- :s3-api:`GetObjectTagging <API_GetObjectTagging.html>`
+- :s3-api:`HeadObject <API_HeadObject.html>`
+- :s3-api:`ListObjects <API_ListObjects.html>`
+- :s3-api:`ListObjectsV2 <API_ListObjectsV2.html>`
+- :s3-api:`ListObjectVersions <API_ListObjectVersions.html>`
+- :s3-api:`PutObject <API_PutObject.html>`
+- :s3-api:`PutObjectTagging <API_PutObjectTagging.html>`
+- :s3-api:`RestoreObject <API_RestoreObject.html>`
+- :s3-api:`SelectObjectContent <API_SelectObjectContent.html>`
+
+Object Locking
+~~~~~~~~~~~~~~
+
+- :s3-api:`GetObjectRetention <API_GetObjectRetention.html>`
+- :s3-api:`PutObjectRetention <API_PutObjectRetention.html>`
+- :s3-api:`GetObjectLegalHold <API_GetObjectLegalHold.html>`
+- :s3-api:`PutObjectLegalHold <API_PutObjectLegalHold.html>`
+- :s3-api:`GetObjectLockConfiguration <API_GetObjectLockConfiguration.html>`
+- :s3-api:`PutObjectLockConfiguration <API_PutObjectLockConfiguration.html>`
+
+Multipart Uploads
+~~~~~~~~~~~~~~~~~
+
+:s3-api:`AbortMultipartUpload <API_AbortMultipartUpload.html>`
+:s3-api:`CompleteMultipartUpload <API_CompleteMultipartUpload.html>`
+:s3-api:`CreateMultipartUpload <API_CreateMultipartUpload.html>`
+:s3-api:`ListMultipartUploads <API_ListMultipartUploads.html>`
+:s3-api:`ListParts <API_ListParts.html>`
+:s3-api:`UploadPart <API_UploadPart.html>`
+:s3-api:`UploadPartCopy <API_UploadPartCopy.html>`
+
+Unsupported API Endpoints
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
```

Comment:
Sphinx is bulleting this list, but not the one for buckets below. Weird.

Maybe try
```suggestion
.. code-block:: text
```